sdks/python/apache_beam/transforms/__init__.py

```diff
@@ -23,3 +23,8 @@
 from apache_beam.transforms.ptransform import *
 from apache_beam.transforms.timeutil import TimeDomain
 from apache_beam.transforms.util import *
+
+try:
+  from apache_beam.transforms.cy_dataflow_distribution_counter import DataflowDistributionCounter
+except ImportError:
+  from apache_beam.transforms.cy_combiners import DataflowDistributionCounter
```

Comment:
<!--new_thread; commit:dd3ae5364c173a99c8a15f7e80960b9307993f7c; resolved:0-->
Is this branch obsolete?  I don't see

Reply:
Sorry I may not 100% understand what you concerns, but the thing i want to do here is:
1. Bring DataflowDistributionCounter into transforms level
2. Try import cython module first, if failed, then import python module. Since we keep the same interfaces, I don't need to have this import pattern everywhere when using this counter.

Reply:
Sorry, this was an incomplete comment; I made a mistake here.